waggle-dance-rest/src/test/java/com/hotels/bdp/waggledance/rest/service/DelegatingFederationServiceTest.java

```diff
@@ -35,9 +35,6 @@
 @RunWith(MockitoJUnitRunner.class)
 public class DelegatingFederationServiceTest {
 
-  private static final String FEDERATED_METASTORE1_URI = "thrift://federatedMetaStore1:9083";
```

Comment:
What's the reason for these changes? They don't seem related to the rest of the PR?

Reply:
I have added them back.